### PR TITLE
Added bsd-simplified

### PR DESCRIPTION
**Pull request by sunilk747**

### CODE REVIEW

File: tests/synonyms/license_synonyms.json [hunk 1]
```diff
@@ -294,7 +300,7 @@
     "ECL 1.0": "unknown", 
     "LGPL with exceptions or ZPL": "unknown", 
     "2-Clause BSD": "unknown", 
-    "BSD-2-Clause": "unknown", 
+    "BSD-2-Clause": "bsd-simplified",
```
Comment (sara-02) [inline tests/synonyms/license_synonyms.json]: Looks like this is a repetition

Comment (sara-02) [inline tests/synonyms/license_synonyms.json]: Looks like this is a repetition.

Comment (sunilk747) [reply tests/synonyms/license_synonyms.json]: Removed.

**Comment by sara-02** [comment]
@LGTM, but looks like you used the wrong remote-branch. Kindly take care of it next time.

**Pull request merged**